deploy/base/model.yaml

```diff
@@ -1 +1 @@
-embeddings_file: embeddings_us_filtered_2023_03_31_19_43_56.csv
+embeddings_file: embeddings_us_filtered_2023_04_04_14_51_24.csv
```

Comment:
should there be an associated update to the CSV file?

Reply:
hm I updated it from the online sheet. i can try to run it again

Reply:
ok great, now the csv shows up and I checked the screenshot prompts: same results :)